pkg/localstore/migration_sharky.go

```diff
@@ -119,6 +120,30 @@ func migrateSharky(db *DB) error {
 		return err
 	}
 
+	db.gcSize, err = db.shed.NewUint64Field("gc-size")
+	if err != nil {
+		return err
+	}
+
+	db.reserveSize, err = db.shed.NewUint64Field("reserve-size")
+	if err != nil {
+		return err
+	}
+
+	gcSize, err := db.gcSize.Get()
+	if err != nil {
+		return err
+	}
+
+	reserveSize, err := db.reserveSize.Get()
+	if err != nil {
+		return err
+	}
+
+	if gcSize+reserveSize > sharkyMaxTotalChunks {
+		return errors.New("maximum allowed chunks exceeded")
```

Comment:
So the answer to my question about what will happen to my massively pinned chunk node is that the sharky migration will fail leaving me with a node that no longer works until I build my own dirty version with a dramatically increased limit.  Ug.

Reply:
Yes you would need to do that. We will try to work around it so that it is not needed. Will keep you in the loop

Reply:
Another option is that we will just up the "limit" (or make it unbounded)